docs/docs/how-tos/How to install and configure Keycloak.md

```diff
@@ -0,0 +1,156 @@
+
+### Add user using Keycloak console
+
+To add a Nebari user from the web console for Keycloak, visit https://{your-nebari-domain}/auth/admin/ and log in using the username `root`, as shown above.
+
+Keycloak uses [realms](https://www.keycloak.org/docs/latest/server_admin/#configuring-realms) to separate their permission scopes under the Keycloak admin console. The realm for Nebari is `nebari` and all Nebari users will be part of the `nebari` realm, on the other hand, the realm for the Keycloak admin console is `master`.
+
+:::note
+The root user alone is a member of the `master` realm. For security best practices, we recommend that you keep at minimum any additional users to the `master` realm. See the [Master realm Keycloak documentation](https://www.keycloak.org/docs/latest/server_admin/#the-master-realm) for more information.
+:::
+
+The `nebari` realm is selected by default, we strongly recommend leaving it as is.
+
+Steps to create a new user:
+
+1. Click **Users** along the left-hand side of the page.
+
+2. Click the **Add user** button and you will see the new user form:
+
+   ![Keycloak add user tab screenshot - new user form](/img/how-tos/keycloak_add_users.png)
+
+3. Fill out the three fields outlined above. These are **Username**, **Email**, and **Groups**. (See explanation below). Then click **save**.
+
+   - Username: Depending on the authentication provider selected ('password', 'GitHub' or 'Auth0'), the values entered into the **Username** field will differ slightly. The following table outlines
+     those differences:
+
+      |          | Password          | GitHub            | Auth0                 |
+      | -------- | ----------------- | ----------------- | --------------------- |
+      | Username | _unique username_ | _GitHub username_ | _Email to login with_ |
+
+- Once the **Username** field is updated, add a valid email address in the **Email** field.
+
+  :::note
+  Although not required, users may not be able to log in to Grafana if this field isn't properly set.
+  :::
+
+- Associate the user with one or more of the **Groups**. Out of the box, Nebari is deployed with the following groups: `admin`, `analyst`, and `developer` (see the [Roles x Groups](#roles-x-groups) section below for more details).
+
+4. Click **Save**.
+
+If you are using the password authentication provider, you will also need to define a password for the user. Keep in mind that, if using Auth0 or Github, the password field is not required. It's best to put the **Temporary** toggle in the **OFF** position. Otherwise the user will be forced to change the password on first login.
+
+![Keycloak add user > credentials tab screenshot - set password](/img/how-tos/keycloak_user_password.png)
+
+### Add user from the command line
+
+:::warning
+If you [changed the initial root password for Keycloak](#change-keycloak-root-password) this method will not work.
+:::
```

Comment:
What is the alternative here? It makes no sense that we are telling people to change their password then making a tool that does not work if they follow security best practices 

this is not good user experience and needs fixing cc @iameskild @viniciusdc @costrouc